django_project/ford3/templates/campus_form.html

```diff
@@ -44,39 +49,38 @@ <h4 class="list-group-item-heading">
                             <div class='row'>
-                                <div class='col-md-5'>
-                                    <h5>All qualifications in {{ campus }} campus</h5>
+                                <div class='col-md-5 campus-all-qualifications'>
+                                    <h5 class="text-center">All qualifications in {{ campus }} campus</h5>
                                     <ul id='campus-qualifications-list'>
                                     </ul>
                                 </div>
-                                <div class='col-md-2 text-center'>
+                                <div class='col-md-2 text-center' style="margin-top: 150px;">
+                                        <p class="qualif-texts">add</p>
                                     <button type='button'
                                             class='left-arrow-button'
                                             data-action='add-qualif'>
-                                        add
                                     </button>
                                     <br />
                                     <br />
                                     <button type='button'
                                             class='right-arrow-button'
                                             data-action='remove-qualif'>
-                                        remove
-                                    </button>                                    
+                                    </button>
+                                    <p class="qualif-texts">remove</p>
 
                                 </div>
                                 <div class='col-md-5'>
                                     <h5>All qualifications</h5>
-                                    <div class='row'>
-                                        <div class='col-md-10'>
-                                            <input type='text' data-action='search-qualif' />
-                                        </div>
-                                        <div class='col-md-2'>
+                                    <div class='row saqa-all-qualifications'>
+                                        <div class='col-md-12' style="box-shadow: 0 2px 2px rgba(0, 0, 0, 0.24), 0 0 2px rgba(0, 0, 0, 0.12);height: 40px">
+                                            <input type='text' data-action='search-qualif' class="search-qualifications" value="Search Qualifications" style="width: 80%;"/>
                                             <button type='button'
                                                     data-action='clear-qualifs'>
                                                 clear
                                             </button>
+
                                         </div>
-                                        <div class='col-md-12'>
+                                        <div class='col-md-12' style="padding-right: 0px;">
```

Comment:
I think there is a Bootstrap class to remove padding. Look at the documentation. Or create a custom class to not use css in html.